code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -114,15 +106,12 @@
 	else
 		cut_overlays()
 
-/obj/item/clothing/head/helmet/space/plasmaman/ComponentInitialize()
-	. = ..()
-	RegisterSignal(src, COMSIG_COMPONENT_CLEAN_ACT, .proc/wipe_that_smile_off_your_face)
-
-///gets called when receiving the CLEAN_ACT signal from something, i.e soap or a shower. exists to remove any smiley faces drawn on the helmet.
-/obj/item/clothing/head/helmet/space/plasmaman/proc/wipe_that_smile_off_your_face()
-	if(smile)
+/obj/item/clothing/head/helmet/space/plasmaman/wash(clean_types)
+	. = ..(clean_types)
+	if(smile && (clean_types & CLEAN_TYPE_PAINT))
 		smile = FALSE
-		cut_overlays()
+		update_icon()
+		. = TRUE
```

Comment:
```suggestion
		return TRUE
```
Return at the end of the proc, instead of assigning the return variable, which implies the proc is not over yet.